### PR TITLE
Update Ci pipeline

### PR DESCRIPTION
**Pull request by dimamo5**

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -4,3 +4,6 @@ codecov:
 
 ignore:
   - "velocidi-sdk/src/main/kotlin/com/velocidi/events"
+
+coverage:
+  range: "50...75"
```
Comment (belerophon) [inline .codecov.yml]: Nitpick: please add a new line here.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -13,13 +13,19 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
+    - uses: actions/cache@v1
```
Comment (belerophon) [inline .github/workflows/ci.yml]: Did this improve compilation times, do you have any sense of the improvements?

File: .codecov.yml (hunk 1 above)
Comment (dimamo5) [reply .codecov.yml]: Sure. Fixed in c5b25985

File: .github/workflows/ci.yml (hunk 2 above)
Comment (dimamo5) [reply .github/workflows/ci.yml]: I think it did. This [branch](https://github.com/velocidi/velocidi-android-sdk/actions?query=branch%3Aupdate-ci++) as lower the run times than [master](https://github.com/velocidi/velocidi-android-sdk/actions?query=branch%3Amaster++). 

The major improvement was in `Build with Gradle` step. Previously, it took ~2:50 sec to run. Now it takes ~1:30 sec.

**Pull request merged**